desktop/dashboard.md

```diff
@@ -1,37 +1,33 @@
 ---
-description: Docker Desktop Dashboard
-keywords: Docker Desktop Dashboard, container view
-title: Docker Desktop Dashboard
+description: Docker Dashboard
+keywords: Docker Dashboard, manage, containers, images
+title: Docker Dashboard
 redirect_from:
 - /docker-for-mac/dashboard/
 - /docker-for-windows/dashboard/
 ---
 
-The Docker Desktop Dashboard provides a simple interface that enables you to interact with containers and applications, and manage the lifecycle of your applications directly from your machine. The Dashboard UI shows all running, stopped, and started containers with their status. It provides an intuitive interface to perform common actions to inspect, interact with, and manage your Docker objects including containers and Docker Compose-based applications.
+The Docker Dashboard provides a simple interface that enables you to manage your containers, applications, and images directly from your machine without having to use the CLI to perform core actions.
 
-The Docker Desktop Dashboard offers the following benefits:
+The **Containers/Apps** view provides a runtime view of all your containers and applications. It allows you to interact with containers and applications, and manage the lifecycle of your applications directly from your machine. This view also provides an intuitive interface to perform common actions to inspect, interact with, and manage your Docker objects including containers and Docker Compose-based applications.
 
-- A GUI to abstract core information from the CLI
-- Access to container logs directly in the UI to search and explore container behavior
-- Access to combined Compose logs from the UI to understand Compose applications
-- Quick visibility into ports being used by containers
-- Monitor container resource utilization
+The **Images** view displays a list of your Docker images, allows you to run an image as a container, pull the latest version of an image from Docker Hub, inspect images, and contains clean up options to remove unwanted images from the disk to reclaim space.
```

Comment:
This sentence has got a bit long and hard to parse. I suggest:

The **Images** view displays a list of your Docker images, and allows you to run an image as a container, pull the latest version of an image from Docker Hub, and inspect images. It also contains clean up options to remove unwanted images from the disk to reclaim space.

Reply:
Although actually it's a bit more than that because it displays your remote images too. So add another sentence about that, something like (please rephrase as required!):

If you are logged in, you can also see the images you and your organization have shared on Docker Hub.

Reply:
Updated